esper/query/management/commands/embed_faces.py

```diff
@@ -0,0 +1,89 @@
+from django.core.management.base import BaseCommand
```

Comment:
You are right. This should be more clear. I haven't changed face_cluster. 
The others just extract embeddings with OpenFace or FaceNet. 
I will rename them